src/build-commands/orb.yml

```diff
@@ -0,0 +1,68 @@
+version: 2.1
+
+description: |
+  A set of commonly used build commands for internal use.
+
+examples:
+  stoplight-build-commands-example:
+    description: "Checkout, set .npmrc, and clone down build scripts."
+
+    usage:
+      version: 2.1
+
+      orbs:
+        stoplight: stoplight/build-commands@0.0.1
+
+      jobs:
+        build:
```

Comment:
Is this job (and commands) run in the same order a lot? If so I you could define it as a job on this orb that you can use such as https://github.com/stoplightio/circleci-orbs/blob/master/src/stoplight-cli/orb.yml#L59. You can also use the individual commands too so it doesn't hurt to add it.

If you need, you can also define pre/post steps (https://circleci.com/docs/2.0/reusing-config/#defining-pre-and-post-steps)

Then the usage example for api-gen3 would be:

```
// example api-gen3 circleci.yml
version: 2.1

orbs:
  stoplight: stoplight/build-commands@0.0.1

workflows:
  version: 2.1
  test_build_deploy:
    jobs:
      - test
      - stoplight/build
           - post-steps
             - run:
                   name: create rpm tarball
                   command: ...
```

Just a thought 😁

Reply:
I don't think I can use these as a "job" since they would then run in their own container. I need them mainly to modify the file system of the currently running container (calling them from within another job)

Reply:
The post-step configuration would work, but then I'd have to put the entire "build-rpm" job into the workflows configuration which is awkward